barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.cpp

```diff
@@ -520,6 +520,8 @@ PairingPointAccumulatorIndices process_avm_recursion_constraints(
     // Add recursion constraints
     size_t idx = 0;
     for (auto& constraint : constraint_system.avm_recursion_constraints) {
+        // TODO(https://github.com/AztecProtocol/barretenberg/issues/1303): Utilize the version of this method that
```

Comment:
Leaving it as a TODO to actually enable in ACIR the new algorithm implemented in this PR.